src/main/java/mate/controller/DeleteDriverController.java

```diff
@@ -0,0 +1,23 @@
+package mate.controller;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.service.DriverService;
+
+@WebServlet(urlPatterns = {"/drivers/delete"})
+public class DeleteDriverController extends HttpServlet {
+    private static final Injector injector = Injector.getInstance("mate");
+    private final DriverService driverService
+            = (DriverService) injector.getInstance(DriverService.class);
+
+    @Override
+    public void doGet(HttpServletRequest req, HttpServletResponse resp) throws IOException {
+        Long id = Long.valueOf(req.getParameter("id"));
+        driverService.delete(id);
+        resp.sendRedirect("/drivers");
```

Comment:
```suggestion
        resp.sendRedirect(req.getContextPath() + "/drivers");
```

add this everywhere. req.getContextPath() can be empty (and that is why it works on your local without this), but it depends on server settings. The code should work with any server settings.